docs/content/users/quickstart.md

```diff
@@ -607,6 +607,66 @@ The Laravel project type can be used for [Statamic](https://statamic.com/) like
     ddev launch /cp
     ```
 
+## Sulu
+
+```bash
+mkdir my-sulu-site && cd my-sulu-site
+ddev config --project-type=php --docroot=public
```

Comment:
```suggestion
ddev config --project-type=php --docroot=public --database=mysql:8.0
```

Reply:
I assumed from your `DATABASE_URL="mysql://db:db@db:3306/db?serverVersion=8.0.32&charset=utf8mb4"
` that mysql 8.0 was somehow important. If it's not, then we can use the defaults.

Reply:
i first had the same assumption but then took a second look at the docs and realized all the standard db ddev supports are supported by sulu as well